doc/tutorials/calib3d/camera_calibration_pattern/camera_calibration_pattern.markdown

```diff
@@ -36,6 +36,10 @@ create a circle board pattern in file acircleboard.svg with 7 rows, 5 columns an
 
         python gen_pattern.py -o acircleboard.svg --rows 7 --columns 5 --type acircles --square_size 10 --radius_rate 2
 
+create a radon checkerboard for [findChessboardCornersSB()](https://docs.opencv.org/master/d9/d0c/group__calib3d.html#gadc5bcb05cb21cf1e50963df26986d7c9) with markers in (7 4), (7 5), (8 5) cells:
```

Comment:
Doxygen should find function by its name, link is not necessary:
```suggestion
create a radon checkerboard for findChessboardCornersSB with markers in (7 4), (7 5), (8 5) cells:
```

Reply:
also need to add parentheses `findChessboardCornersSB`, this done